pubtools/_quay/item_processor.py

```diff
@@ -79,8 +86,10 @@ def __init__(
         self.sleep_time: int = sleep_time
         self.timeout: int = timeout
         self.poll_rate: int = poll_rate
+        self.full_extract = full_extract
+        self.manifest_cache = {}
```

Comment:
Added log to check how manifest_cache is used:
```
diff --git a/pubtools/_quay/item_processor.py b/pubtools/_quay/item_processor.py
index 8d061e2..5879878 100644
--- a/pubtools/_quay/item_processor.py
+++ b/pubtools/_quay/item_processor.py
@@ -223,6 +223,7 @@ class ContentExtractor:
         for mtype in mtypes:
             ret_headers = {}
             if self.manifest_cache.get(image_ref, {}).get(mtype):
+                LOG.error("Get cache: %s, %s" % (image_ref, mtype))
                 manifest, ret_headers = self.manifest_cache[image_ref][mtype]
                 if manifest:
                     mret: ManifestArchDigest | List[ManifestArchDigest] = MEDIA_TYPES_PROCESS[mtype](
@@ -263,6 +264,7 @@ class ContentExtractor:
                     break
                 time.sleep(self.sleep_time)
             self.manifest_cache.setdefault(image_ref, {})[mtype] = (manifest, ret_headers)
+            LOG.error("Set cache: %s, %s" % (image_ref, mtype))
```
There's only "Set cache" but no "Get cache":
```
# curl -s http://10.0.150.153:32771/pub/task/2472/log/stdout.log | grep  "Get cache"
# curl -s http://10.0.150.153:32771/pub/task/2472/log/stdout.log | grep  "Set cache" | wc -l
11
```
So I doubt cache improved anything. Could you remove it?